roles/p100/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+cds_ip: 10.3.3.207
```

Comment:
if we put this as a var in the `./vars` we can use it here without needing to overwrite it everywhere.

Reply:
wouldn't vars override the defaults ?

Reply:
i'm thinking if we wind up using 2 cds's,  either wf46 or wf47 would get a vars for cds_ip to change it's ip.
and per https://docs.ansible.com/ansible/2.9/user_guide/playbooks_variables.html#playbooks-variables  that is the preferred way to do it.